orange/src/main/java/com/hartwig/hmftools/orange/OrangeWgsConfig.java

```diff
@@ -0,0 +1,193 @@
+import static com.hartwig.hmftools.common.pipeline.PipelineToolDirectories.METRICS_DIR;
+import static com.hartwig.hmftools.common.pipeline.PipelineToolDirectories.PEACH_DIR;
+import static com.hartwig.hmftools.common.pipeline.PipelineToolDirectories.SAGE_GERMLINE_DIR;
+import static com.hartwig.hmftools.common.pipeline.PipelineToolDirectories.SAGE_SOMATIC_DIR;
+import static com.hartwig.hmftools.common.pipeline.PipelineToolDirectories.SIGS_DIR;
+import static com.hartwig.hmftools.common.pipeline.PipelineToolDirectories.VIRUS_INTERPRETER_DIR;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.CHORD_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.CHORD_DIR_DESC;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.CUPPA_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.CUPPA_DIR_DESC;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.LINX_GERMLINE_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.PEACH_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.PEACH_DIR_DESC;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.PIPELINE_SAMPLE_ROOT_DIR;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.SAGE_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.SAGE_GERMLINE_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.SAMPLE_DATA_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.SIGS_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.SIGS_DIR_DESC;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.VIRUS_DIR_CFG;
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.VIRUS_DIR_DESC;
+import static com.hartwig.hmftools.common.utils.file.FileWriterUtils.checkAddDirSeparator;
+import static com.hartwig.hmftools.orange.OrangeApplication.LOGGER;
+import static com.hartwig.hmftools.orange.OrangeConfig.TUMOR_SAMPLE_ID;
+import static com.hartwig.hmftools.orange.OrangeConfig.getMetricsFile;
+import static com.hartwig.hmftools.orange.OrangeConfig.getToolDirectory;
+import static com.hartwig.hmftools.orange.util.Config.fileIfExists;
+import static com.hartwig.hmftools.orange.util.Config.optionalFileIfExists;
+
+import com.hartwig.hmftools.common.chord.ChordDataFile;
+import com.hartwig.hmftools.common.cuppa.CuppaDataFile;
+import com.hartwig.hmftools.common.sage.SageCommon;
+import com.hartwig.hmftools.common.sigs.SignatureAllocationFile;
+import com.hartwig.hmftools.common.utils.config.ConfigBuilder;
+import com.hartwig.hmftools.common.virus.AnnotatedVirusFile;
+
+import org.immutables.value.Value;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Value.Immutable
+@Value.Style(passAnnotations = { NotNull.class, Nullable.class })
+public interface OrangeWgsConfig
```

Comment:
The term `Wgs` as well as `WGS` exist within hmftools. In ORANGE, this concept is always called `WGS` so far so might be good to remain consistent? Also I think I would add "Ref" in the name somewhere, to make it clearer this is the WGSRefConfig!

Reply:
Ok. Note we did end up maintaining the WGS tumor only possibility, at least to the level it was previously supported which may have been just partial.